pkg/collector/corechecks/snmp/internal/metadata/store_test.go

```diff
@@ -72,6 +73,11 @@ func TestStore_Column(t *testing.T) {
 	assert.ElementsMatch(t, []string{"1", "2", "3"}, store.GetColumnIndexes("interface.admin_status"))
 	assert.Equal(t, []string{"3"}, store.GetColumnIndexes("interface.oper_status"))
 	assert.Equal(t, []string(nil), store.GetColumnIndexes("interface.does_not_exist"))
+
+	// test GetColumnAsByteArray
+	assert.Equal(t, []byte{0x1, 0x2, 0x3, 0x4, 0x5, 0x6}, store.GetColumnAsByteArray("interface.mac_address", "1"))
+	assert.Equal(t, []byte(nil), store.GetColumnAsByteArray("interface.mac_address", "2"))
+	assert.Equal(t, []byte(nil), store.GetColumnAsByteArray("interface.does_not_exist", "1"))
```

Comment:
Since the function being tested also supports strings, should we have a test case for this as well?

Reply:
@ken-schneider Good point, I think we can probably only support byte array for the moment.
So, instead of changing the test, I removed support for `string`. 

Here is the change ba9f5b2e9bad9a17b904faaf4c867ef2a01a141d

Let me know if it's ok for you.